docs/guides/integration-mobile-overview.md

```diff
@@ -58,6 +49,108 @@ For details, refer to one of the following guides:
 - [Client-Side Integration Guide for Mobile](integration-mobile-client-side.md)
 - [Client-Server Integration Guide for Mobile](integration-mobile-client-server.md)
 
+## (NEW SECTION STARTS HERE)
+
+## Mobile Integration Paths
+
+To determine the best integration path for your mobile scenario, consider these points:
+
+1. Do you want to retrieve the UID2 token on the client side or on the server side? See [Client-Side or Client-Server Integration](#client-side-or-client-server-integration).
+
+1. What do you want to use to retrieve and refresh the UID2 token? See [Paths to Manage the UID2 Token](#paths-to-manage-the-uid2-token).
+
+1. How do you want to use the UID2 tokens? See [Paths to Send the Token to the Bidstream](#paths-to-send-the-token-to-the-bidstream).
+
+### Client-Side or Client-Server Integration?
+
+The options for integrating with UID2 using the UID2 mobile SDKs are summarized in the following table. Choose the <Link href="../ref-info/glossary-uid#gl-integration-approaches">integration approach</Link> that's best for you.
+
+| Scenario | Option | Integration Guide |
+| :--- | :--- | :--- |
+| You have access to <Link href="../ref-info/glossary-uid#gl-dii">DII</Link> (email address or phone number) on the client side/within the mobile app, and want to keep changes within your app only. | Client-side integration | [UID2 Client-Side Integration Guide for Mobile](integration-mobile-client-side.md) |
+| You have access to DII on the server side only and can do the necessary development to generate UID2 tokens server-side, or you are using a <Link href="../ref-info/glossary-uid#gl-private-operator">Private Operator</Link>. | Client-Server Integration | [UID2 Client-Server Integration Guide for Mobile](integration-mobile-client-server.md) |
+
+### Managing the UID2 Token
```

Comment:
it may be better to mention what managing UID2 Token means - "To create, store and refresh tokens"

Reply:
OK. Done.